2a_model/src/models/config_base.yml

```diff
@@ -10,47 +10,41 @@ recurrent_dropout: 0.2
 finetune_learning_rate: 0.01
 early_stopping: False
 validation_sites:
-  - '01472104'
-  - '01473500'
-  - 01481500
-test_sites:
-  - '01475530'
-  - 01475548
-train_start_date: '1980-01-01'
-train_end_date: '2014-10-01'
-val_start_date: '2014-10-01'
-val_end_date: '2015-10-01'
-test_start_date: '2015-10-01'
-test_end_date: '2022-10-01'
+  - "01475530"
+  - "01475548"
+train_start_date: '1980-10-01'
+train_end_date: '2015-10-01'
```

Comment:
It looks like from `p2a_site_splits` that records on `2015-10-01` are treated as validation data, is that right?

Reply:
Yes.